### DISCUSSION TITLE
Just a sincere thank you.

### DISCUSSION THREAD

**Discussion by bboehs** [LesFerch/WinSetView#82] [General]
Hi,
I wish I'd found WinSetView sooner. The myriad list of dumb windows issues never seems to never seems to grow smaller. Thankfully there are great people such as yourself who see them and do something. 

Somewhere along the line, MS forgot that computers are "supposed" to make life easier and even though we can do things that weren't possible 50 years ago, or even a year or two, The world needs more people like you to make a dent in shortening that list.

Kudos for a nice simple tool that demonstrates "easier" is still possible! 

Cheers,
B

**Comment by LesFerch** [comment]
You're very welcome and thank you! 😊

**Comment by Zhurama** [comment]
OMGOMGOMGOMGOMG!!! Thank YUUUU!! This nagging niggly problem with windows has totally annoyed me since it began. Bless you for this tool.. It really made my day.

Namaste

**Reply by LesFerch** [reply]
Your message made my day. 🙂

**Comment by buckwilicker** [comment]
Thank you LesFerch. This made managing my media server so much easier. Windows 11 is truly a headache for these settings compared to Windows 10.

**Comment by hkpD** [comment]
Hello, just tacking on another "thank you" to the developer of this app. 

After hours of searching, i found this app and resolved my explorer view issues

Very grateful for the creator of this app!

**Comment by FLyBoyMcCall** [comment]
Thank you very much for this simple time saver.

**Comment by TaskMetadata** [comment]
12am last night, I had made [this](https://aka.ms/AAz9ei8) windows feedback post... as I was about to give up for the night, I saw [a post](https://learn.microsoft.com/en-us/answers/questions/4365023/apply-view-format-to-all-sub-folders) that mentioned this... Gave it a try today, I'm so glad I did! it works!

I'm really not sure why windows doesn't allow you to apply the view to all folders within a library... it means you have to set every single fodler individually haha. I'm not doing that!

Thank you very much for this!

P.s. either I'm misunderstanding, or... the option to set a different view for save dialogs... I can can select details, but then there's no column button for it?

P.p.s. Also, I appreciate all of the little extra stuff, like the windows search highlights and internet search. Not everyone will use them, but they're useful to others!

**Reply by LesFerch** [reply]
The different view for dialogs does not have an option to have a different set of columns, but it will use the column settings for the main view. For example, let's say you have the Pictures folder type set to Large icons. You can still click the Columns button and configure which columns are displayed if you were to change to Details view and those same columns apply to file dialog views as well.
